### ISSUE TITLE
At client startup, report the run with the best validation loss

### ISSUE THREAD

**Issue by kostaleonard** [kostaleonard/adjutant#4]
When the client starts up, the startup message on Discord should contain not only the number of runs found for the project, but also the summary (train/val loss curve, lowest train/val loss, link to run page on WandB) for the run with the lowest validation loss.
